libs/utils/lana.js

```diff
@@ -1,20 +1,86 @@
-/*
- * Copyright 2022 Adobe. All rights reserved.
- * This file is licensed to you under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License. You may obtain a copy
- * of the License at http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software distributed under
- * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
- * OF ANY KIND, either express or implied. See the License for the specific language
- * governing permissions and limitations under the License.
- */
-
-/**
- * LOG ALWAYS; NEVER ASSUME
- * @param {String} msg the message you want to log
- */
-export default function log(msg) {
-  // eslint-disable-next-line no-console
-  console.log(msg);
+var MSG_LIMIT = 2000;
+
+var defaultOptions = {
+  clientId: '',
+  debug: false,
+  endpoint: 'https://www.adobe.com/lana/ll',
+  endpointStage: 'https://www.stage.adobe.com/lana/ll',
+  errorType: 'e',
+  sampleRate: 1,
+  implicitSampleRate: 1,
+  useProd: true,
+};
+
+function setClientdId(id) {
```

Comment:
Was this supposed to be exposed with an extra d after client? (`setClientdId`)  @chrischrischris

Reply:
Doh!  Fixed with https://github.com/adobecom/milo/pull/26